uPortal-webapp/src/main/webapp/media/skins/respondr/common/less/footer.less

```diff
@@ -20,6 +20,11 @@
 /* ==========================================================================
    Footer Nav
    ========================================================================== */
+#sitemap-tab-header {
```

Comment:
There's probably a better place for this style rule.  It's portlet-specific, so should probably go into `skins/respondr/common/less/views`.